daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Node.scala

```diff
@@ -338,6 +344,44 @@ object Node {
 
   }
 
+  final case class NodeRollback[+Nid, +Cid](
+      // TODO https://github.com/digital-asset/daml/issues/8020
+      // Figure-out what information needs to be contained in a Rollback node.
+      // For the moment, we just have the children.
+      children: ImmArray[Nid]
+  ) extends GenNode[Nid, Cid]
+      with NodeInfo.Rollback {
+
+    def templateId: TypeConName =
```

Comment:
```suggestion
    override def templateId: TypeConName =
```

Reply:
Added the `override`s -- I'm happy to be consistent within this file
Maybe one day I'll understand why it matters :shrug: